app/commands.py

```diff
@@ -832,3 +842,147 @@ def purge_csv_bucket():
     print("ABOUT TO RUN PURGE CSV BUCKET")
     s3.purge_bucket(bucket_name, access_key, secret, region)
     print("RAN PURGE CSV BUCKET")
+
+
+"""
+Commands to load test data into the database for
+Orgs, Services, Users, Jobs, Notifications
+
+faker is used to generate some random fields. All
+database commands were used from tests/app/db.py
+where possible to enable better maintainability.
+"""
+fake = Faker(["en_US"])
```

Comment:
With this being an initialization line for the `Faker` library, I would put it at the top of the file after all of the imports instead (so around line 76 or so, give or take a line given `isort`'s and `black`'s formatting preferences).

`fake` is essentially a variable being used as a constant throughout the rest of file, and toward the top after all of the imports are where such things get set following standard Python style guidelines (mainly [PEP-8](https://peps.python.org/pep-0008/)). 🙁

Reply:
Added to line 77 with comments on 76